lib/git/status_shortcuts.sh

```
@@ -281,22 +281,23 @@ git_commit_prompt() (
-  GIT_PIPE_STATUS=("${PIPESTATUS[@]}${pipestatus[@]}")
-  if breeze_shell_is "zsh"; then
-    git_exit_status="${GIT_PIPE_STATUS[2]}" # zsh array indexes start at 1
-  else
-    git_exit_status="${GIT_PIPE_STATUS[1]}"
-  fi
-  if [[ "$git_exit_status" == 0 ]]; then
-    # Delete saved commit message if commit was successful
-    rm -f "$commit_msg_file"
-  fi
-)
+    local GIT_PIPE_STATUS=("${PIPESTATUS[@]}${pipestatus[@]}")
```

Shellcheck (0.11.0)

[warning] pipestatus is referenced but not assigned.

(SC2154)